uiowa/uiowa
# Added initial back to top functionality

Closes #2755 

NOTE: Removed ability to change node by node - hoping that helps make it less confusing for users. If we want to add that functionality down the road, at least we have this PR to look back at and can easily implement it if we want to go that route.
## Testing Instructions

1. `git checkout scroll-to-top`
2. `blt ds --site=sandbox.uiowa.edu`
3. `yarn workspace uids_base gulp --development; drush @sandbox.local cr`
4. `drush @sandbox.local uli`
5. Click through pages. Make sure no 'back to top' button is visible
6. go to https://sandbox.local.drupal.uiowa.edu/admin/appearance/settings/uids_base & check the checkbox next to 'Add Scroll to top button' & save page
6. Now click through pages - make sure look at an article and person content type and make sure displays there too.
